templates/containers/what-are-containers.html

```diff
@@ -0,0 +1,250 @@
+{% extends "containers/base_containers.html" %}
+
+{% block title %}What are containers{% endblock %}
+
+{% block meta_description %}What are containers and how do containers enable fast-moving modern software, from the cloud to the edge? Learn how to use Ubuntu from the host kernel to container images running cloud applications to get a stable, secure, and consistent developer experience with Docker, Kubernetes, and LXD.{% endblock meta_description %}
+
+{% block meta_image %}https://assets.ubuntu.com/v1/fece8a11-container-in-the-cloud.svg{% endblock meta_image%}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1hStuIzY8dDdFEW4Q0TwboOx0_-SB5klk8zRuNIn3V4o/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-bottomed">
+    <div class="row u-equal-height">
+      <div class="col-7">
+        <h1>What are containers?</h1>
+        <p>Containers are a virtualisation technology that isolates each process in a lightweight and standard environment. They allow users to package and quickly deploy entire applications parallel to each other on the same kernel and hardware, while maintaining isolation among the workloads.</p>
+      </div>
+      <div class="col-5 u-align--center u-vertically-center u-hide-small">
```

Comment:
I think two dashes are needed here! :)
```suggestion
      <div class="col-5 u-align--center u-vertically-center u-hide--small">
```